### DISCUSSION TITLE
Whether time and timezone synchronization is required between the Fluentbit device and the OpenObserve machine to receive data successfully ?

### DISCUSSION THREAD

**Discussion by Poorneshmindteck** [openobserve/openobserve#1601] [General]
Is it mandatory for the date, time, and timezone to be synchronized between the device running Fluentbit (SoC: i.MX6UL with Yocto OS) and the machine running OpenObserve (Ubuntu) in order to successfully receive data on OpenObserve? The issue observed is that Fluentbit on the remote device displays a message stating that only the last 5 hours of data can be ingested. Here are the current time and timezone configurations:

On Ubuntu:
- Local time: Wed 2023-10-04 09:53:17 IST
- Universal time: Wed 2023-10-04 04:23:17 UTC
- RTC time: Wed 2023-10-04 04:23:18
- Time zone: Asia/Kolkata (IST, +0530)

On i.MX6UL:
- Local time: Tue 2023-10-03 06:01:36 UTC
- Universal time: Tue 2023-10-03 06:01:36 UTC
- RTC time: Tue 2023-10-03 06:01:36
- Time zone: UTC (UTC, +0000)

**Comment by oasisk** [comment]
Openobserve by default allows data stale upto 5 hours (UTC now - 5 hours), but you can change it by setting appropriate value for env variable `ZO_INGEST_ALLOWED_UPTO`

**Comment by Poorneshmindteck** [comment]
Thank you, @oasisk, for your response. I was able to resolve the issue with Fluentbit by configuring the "ZO_INGEST_ALLOWED_UPTO" environment variable. However, I encountered another problem when trying to visualize the same data in Openobserve , the data was not displaying. When I synchronized the date and time between the Ubuntu machine running Openobserve and Fluentbit running device , then I was able to visualize the data in Openobserve.

Is it necessary to synchronize the date and time between the Fluentbit device and the Openobserve machine in order to visualize the data??

**Reply by oasisk** [reply]
Please note openobseve stores data in UTC as timestamp , since you ingested stale data, it would available at older timestamp , request you to select appropriate time window to search for data.
